### PR TITLE
Add configuration

### PR DESCRIPTION
**Pull request by DanielMwendwa**

### CODE REVIEW

File: code_schemes/ws_correct_dataset.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+    "SchemeID": "Scheme-fffffb5c",
+    "Name": "WS - Correct Dataset",
+    "Version": "0.0.0.1",
+    "Codes": [
+        {
```
Comment (IsaackMwenda) [inline code_schemes/ws_correct_dataset.json:6]: This should contain all the ws codes in ke vax plus the new datasets for this project.

File: configurations/pipeline_config.json [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  "DriveUpload": {
+    "DriveCredentialsFileURL": "gs://avf-credentials/pipeline-runner-service-acct-avf-data-core-64cc71459fe7.json",
+    "ProductionUploadPath": "ke_constitution_review_analysis_outputs/production.csv",
+    "MessagesUploadPath": "ke_constitution_review_analysis_outputs/messages.csv",
+    "IndividualsUploadPath": "ke_constitution_review_analysis_outputs/individuals.csv",
+    "AutomatedAnalysisDir": "ke_constitution_review_analysis_outputs/automated_analysis"
+  },
+  "MemoryProfileUploadURLPrefix": "gs://avf-pipeline-logs-performance-nearline",
+  "DataArchiveUploadURLPrefix": "gs://pipeline-execution-backup-archive",
+  "BucketDirPath": "2021/KE-VAX"
```
Comment (IsaackMwenda) [inline configurations/pipeline_config.json]:
```suggestion
  "BucketDirPath": "2021/KE-CONSTITUTION-REVIEW"
```

File: configurations/pipeline_config.json [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    {
+      "SourceType": "GCloudBucket",
+      "ActivationFlowURLs": [],
+      "SurveyFlowURLs": [
+        "gs://avf-project-datasets/2021/Kenya-Pool/covid19_ke_urban_s01_demog.jsonl",
+        "gs://avf-project-datasets/2021/Kenya-Pool/covid19_s01_demog.jsonl",
+        "gs://avf-project-datasets/2021/Kenya-Pool/oxfam_wash_s01_demog.jsonl",
+        "gs://avf-project-datasets/2021/Kenya-Pool/undp_kenya_s01_demog.jsonl",
+        "gs://avf-project-datasets/2021/Kenya-Pool/unicef_covid19_ke_s01_demog.jsonl",
+        "gs://avf-project-datasets/2021/Kenya-Pool/worldvision_s01_demog.jsonl"
```
Comment (IsaackMwenda) [inline configurations/pipeline_config.json]: add gpsdd demog  datasets
"gs://avf-project-datasets/2021/Kenya-Pool/gpsdd_bungoma_s01_demog.jsonl",
"gs://avf-project-datasets/2021/Kenya-Pool/gpsdd_kilifi_s01_demog.jsonl",
"gs://avf-project-datasets/2021/Kenya-Pool/gpsdd_kiambu_s01_demog.jsonl"

File: configurations/pipeline_config.json [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  "ListeningGroupCSVURLs": [],
+  "PhoneNumberUuidTable": {
+    "FirebaseCredentialsFileURL": "gs://avf-credentials/avf-id-infrastructure-firebase-adminsdk-6xps8-b9173f2bfd.json",
+    "TableName": "KE-VAX_phone_number_avf_phone_id"
+  },
+  "OperationsDashboard": {
+    "FirebaseCredentialsFileURL": "gs://avf-credentials/avf-dashboards-firebase-adminsdk-gvecb-ef772e79b6.json"
+  },
+  "RapidProKeyRemappings": [
+    {"RapidProKey": "avf_phone_id", "PipelineKey": "uid"},
```
Comment (IsaackMwenda) [inline configurations/pipeline_config.json:49]: add demog RapidProKeyRemappings reference engagement poll https://github.com/AfricasVoices/Project-KE-VAX/blob/main/configurations/engagement_poll_pipeline_config.json

**Review by IsaackMwenda — CHANGES_REQUESTED** [review]

File: configurations/pipeline_config.json (hunk 3 above)
Comment (DanielMwendwa) [reply configurations/pipeline_config.json]: updated, thanks.

File: configurations/pipeline_config.json (hunk 4 above)
Comment (DanielMwendwa) [reply configurations/pipeline_config.json:49]: Updated, thanks.

File: configurations/pipeline_config.json [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+
+    {"RapidProKey": "appointment_views (Text) - ke_constitution_review_poll", "PipelineKey": "appointment_views_raw", "IsActivationMessage": true},
```
Comment (IsaackMwenda) [inline configurations/pipeline_config.json]:
```suggestion
    {"RapidProKey": "Appointment_Views (Text) - ke_constitution_review_poll", "PipelineKey": "appointment_views_raw", "IsActivationMessage": true},
```
The keys are usually Snake cased even though the variable names are not

**Review by IsaackMwenda — APPROVED** [review]
This looks good now thanks @DanielMwendwa

**Pull request merged**